dali/operators/random/rng_base_gpu.cuh

```diff
@@ -126,13 +202,13 @@ void RNGBase<Backend, Impl, IsNoiseGen>::RunImplTyped(workspace_t<GPUBackend> &w
   gridDim.x = 1;
   gridDim.y = div_ceil(blockdesc_count, blockDim.y);
 
-  if (use_default_dist) {
-    RNGKernel<T, Dist, true>
-      <<<gridDim, blockDim, 0, ws.stream()>>>(blocks_gpu, rngs, nullptr, blockdesc_count);
-  } else {
-    RNGKernel<T, Dist, false>
-      <<<gridDim, blockDim, 0, ws.stream()>>>(blocks_gpu, rngs, dists, blockdesc_count);
-  }
+  VALUE_SWITCH(use_default_dist ? 1 : 0, DefaultDist, (false, true), (
```

Comment:
A random thought: perhaps we should have a BOOL_SWITCH and get rid of this `default:` statement nonsense.

Reply:
Agree. Let's handle that in a separate PR.